app/views/cookie_preferences/show.html.erb

```diff
@@ -27,27 +27,27 @@
         <tbody class="govuk-table__body">
           <tr scope="row" class="govuk-table__row">
             <td class="govuk-table__cell "><span class="break-word"><%= Settings.cookies.session.name %></span></td>
-            <td class="govuk-table__cell ">Remembers your identity when you are signed in</td>
-            <td class="govuk-table__cell ">After 6 hours or on signing out</td>
+            <td class="govuk-table__cell ">Stores encrypted session data</td>
+            <td class="govuk-table__cell ">When you close your browser</td>
           </tr>
           <tr scope="row" class="govuk-table__row">
-            <td class="govuk-table__cell"><%= Settings.cookies.consent.name %></td>
+            <td class="govuk-table__cell"><%= Settings.cookies.analytics.name %></td>
             <td class="govuk-table__cell">Saves your cookie consent settings</td>
             <td class="govuk-table__cell">6 months</td>
           </tr>
         </tbody>
       </table>
 
-      <h2 class="govuk-heading-m">Measuring website usage (Google Analytics)</h2>
+      <h2 class="govuk-heading-m">Cookies that measure website use (analytics)</h2>
 
-      <p class="govuk-body">With your permission, we use Google Analytics to collect data about how you use Publish. This information helps us improve our service.</p>
+      <p class="govuk-body">With your permission, we use Google Analytics to collect data about how you use ‘<%= t("service_name.publish") %>’. This information helps us improve our service.</p>
```

Comment:
Revert back to original: With your permission, we use Google Analytics to collect data about how you use Publish. This information helps us improve our service.